### ISSUE TITLE
[css-scrollbars-1] Clarify contrast expectations

### ISSUE THREAD

**Issue by fantasai** [w3c/csswg-drafts#2968]
What happens if an author specifies `scrollbar-face-color: black; scrollbar-track-color: black`? Is that expected to do something reasonable or unreasonable? Does the UA perform fixup? Is the scrollbar expected to have borders or other aspects which would provide sufficient contrast? Is it expected to be a WCAG violation?

**Comment by upsuper** [comment]
If author explicitly specify this kind of thing, we should just show as is.

It's not different to someone setting `color: black; background: black;`, is it?

There were concerned raised in our Intent to Implement that what should be done when only one of the color is specified (and the other is left auto). That case should probably be considered.

**Comment by yurikhan** [comment]
Sorry for being late to this discussion.

The current wording after #3113 calls for “enough contrast” and links to Techniques G183 which calls for a contrast ratio of 3:1 or above, presumably in analogy with contrast recommendations for large text.

Now, I’d like to provide a data point. As a user/reader/developer, I find myself distracted by any contrasting spot of color that I see in my peripheral vision. A scrollbar thumb, 15px wide and having 3:1 contrast against the scrollbar track, and possibly more than that against the page background, certainly qualifies.

(I did a mockup in GIMP. A stroke width of 15px roughly corresponds to the letter I at 144pt font size. That is 8 times the definition of Large-scale text!)

As another data point, on my GTK+3/X11/GNU/Linux desktop, I am using a customized Clearlooks-Phenix theme. Specifically, I set the thumb color at `#e3e4e2`. With the default track color of `#d6d4d2`, it makes for a contrast ratio of 1.15. At this contrast ratio, the scrollbar is not distracting, and is clearly discernible for me when I’m looking straight at it.

Now, I cannot claim to be a real accessibility expert; after all, I’m only −5 diopters myopic and wear corrective lenses at all times. But I suggest that there may be a non-negligible category of users for whom web sites following the 3:1 scrollbar contrast recommendation will prove extremely annoying.
